src/lucky_record/model.cr

```diff
@@ -9,6 +9,13 @@ class LuckyRecord::Model
   macro inherited
     FIELDS = [] of {name: Symbol, type: Object, nilable: Bool, autogenerated: Bool}
     ASSOCIATIONS = [] of {name: Symbol, foreign_key: Symbol, type: Object}
+    HAS_MANY = [] of {name: Object, foreign_key: Symbol, type: Object, through: Object}
```

Comment:
Definitely not loving storing the has_many associations one more time, but I think I need it. Do you see a cleaner way?

Reply:
You could potentially store the association type in the ASSOCIATIONS constant. `{relationship_type: :has_many|:belongs_to|:has_one|:etc}`

Reply:
I did so, thanks. Tiny bit cleaner I think.